templates/network/resources/vkcs_networking_subnet.md.tmpl

```diff
@@ -11,7 +11,7 @@ description: |-
 {{ .Description }}
 
 ## Example Usage
-{{tffile .ExampleFile}}
+{{tffile "examples/networking/subnet/main.tf"}}
```

Comment:
ditto

Reply:
yes, this is changed back to .ExampleFile according to info i got from Stanislav